Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN mvn clean install -DskipTests
 # RUN THE APPLICATION
 #
 FROM openjdk:18-ea-bullseye
-RUN apt-get update & apt-get upgrade
+RUN apt-get update & apt-get upgrade -y
```

Comment:
Still not convinced we need this line.. I actually already removed it from either the server or LDI.. We shouldn't need to update an image we pull from.